e2e/tests/components/radiobuttons/radiobuttons.po.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { browser, element, by } from 'protractor';
+
+export class RadioButtonsPage {
+        
+    getPage(): void {
+        browser.get('/radiobuttons');
+    }
+    
+    titleText = browser.getTitle();
+    
+    radiobutton1 = element(by.id('radiobutton1'));
+    radiobutton2 = element(by.id('radiobutton2'));
+    radiobutton3 = element(by.id('radiobutton3'));
+    radiobutton4 = element(by.id('radiobutton4'));
+    text1 = element(by.id('text1'));
+    disableFirstButton = element(by.id('button1'));
+    changeToSimplified = element(by.id('button2'));
+    
+    confirmIsChecked = function(radiobutton: any) {

```

Comment:
Add types to the function parameters so we get error detection in VS Code. Correct type for this is `ElementFinder` I believe